running_local_operator.md

```diff
@@ -0,0 +1,46 @@
+# Running a local copy of an operator for development without conflicts
+
+You can run a local copy of an operator for development using commands
+like the following in that operator's directory.
+```
+make manifests generate build
+OPERATOR_TEMPLATES=$PWD/templates ./bin/manager
```

Comment:
could be `OPERATOR_TEMPLATES=$PWD/templates make run` but that's not the goal for this PR